docs/develop/go/message-passing.mdx

```diff
@@ -316,12 +316,20 @@ if err != nil {
 
 ### Send an Update {#send-update-from-client}
 
+:::tip Stability
+
+In [Pre-release](/evaluate/development-production-features/release-stages#pre-release) (API is subject to change).
```

Comment:
This needs to be updated to Public Preview. We do not release docs (as a rule) before then, so it should reflect the status when this material is ready for going live.